components/FormulaireUnicorn.vue

```diff
@@ -74,12 +84,18 @@ export default{
                 weight: this.unicorn?.weight || '',
                 hobbies: this.unicorn?.hobbies || '',
                 photo: this.unicorn?.photo || null
-            }
+            },
+
+            image: ''
         }
     },
+
     methods: {
         actSubmit() {
             this.$emit("act", this.form)
+        },
+        editImage(e){
+            this.image = URL.createObjectURL(e.target.files[0])
         }
```

Comment:
si tu mets ici (ligne 98) this.image = e.targent.value, ça ne fonctionne pas ? Ou alors this.image = e.target.files[0] ?

Reply:
les deux ne fonctionnent pas